docs/user-guide/knowledge-distillation.rst

```diff
@@ -1,5 +1,9 @@
 .. include:: /content/nemo.rsts
 
+.. include:: aligner-algo-header.rst
+
+.. _nemo-aligner-knowledge-distillation:
+
```

Comment:
Line 18/22 use capitalization in title

Obtain the Fine-Tuned Teacher and Pre-Trained Student Models

Line 20/24 fix punctuation

To start, we must first download both the pre-trained student and fine-tuned teacher models.

Line 45/49 fix punctuation

After these steps, you should have files ``2b_student.nemo`` and ``teacher_checkpoint/Nemotron-3-8B-Chat-4k-SFT.nemo`` to use in NeMo-Aligner.